src/extlibs/hrvo/kd_tree.cpp

```diff
@@ -42,14 +42,14 @@ KdTree::KdTree(HRVOSimulator *simulator) : simulator_(simulator) {}
 
 void KdTree::build()
```

Comment:
This whole file should be removed in favour of using: https://github.com/UBC-Thunderbots/Software/blob/master/src/software/geom/algorithms/nearest_neighbor_search.hpp